app/recurring_job.go

```diff
@@ -56,7 +46,7 @@ type Job struct {
 	volumeName   string
 	snapshotName string
 	retain       int
-	jobType      string
+	task         longhorn.RecurringJobType
 	labels       map[string]string
```

Comment:
nit: not sure why changing the field name, but I would suggest the naming is based on the type, so here `jobType` is quite suitable.

Reply:
This is actually optional, so if decide not to change, it's also fine.

Reply:
Feel this should be consistent with the [RecurringJobSpec.Task](https://github.com/longhorn/longhorn-manager/blob/master/k8s/pkg/apis/longhorn/v1beta2/recurringjob.go#L38).

Reply:
the spec has been defined, so this is why I said is nice to change. Better to keep it as is to make things compatible first.